sqlframe/base/session.py

```diff
@@ -202,13 +203,16 @@ def range(self, *args):
 
     def createDataFrame(
         self,
-        data: t.Sequence[
-            t.Union[
-                t.Dict[str, ColumnLiterals],
-                t.List[ColumnLiterals],
-                t.Tuple[ColumnLiterals, ...],
-                ColumnLiterals,
-            ]
+        data: t.Union[
+            t.Sequence[
+                t.Union[
+                    t.Dict[str, ColumnLiterals],
+                    t.List[ColumnLiterals],
+                    t.Tuple[ColumnLiterals, ...],
+                    ColumnLiterals,
+                ],
+            ],
+            "pd.DataFrame",
```

Comment:
The quotes shouldn't be needed here because of this line at the top: `from __future__ import annotations`